src/wyzeapy/utils.py

```diff
@@ -72,12 +72,13 @@ def create_password(password: str) -> str:
     return hashlib.md5(hex2.encode()).hexdigest()
 
 
-def check_for_errors_standard(response_json: Dict[str, Any]) -> None:
+def check_for_errors_standard(call, response_json: Dict[str, Any]) -> None:
```

Comment:
I'd probably typehint "call" so that it's clearer what call is.

Reply:
Might be a dumb question, but is there any easy way to typehint this since its going to be passed an instance of a bunch of different classes? Or maybe I can just make the param name more clear..

Reply:
I'm looking at this on my phone right now.... But does it get passed a bunch of different types of classes? I could've sworn it only ever calls these from the authlib class? I don't think anything else makes any http calls.

Making the name clearer would probably help as well. This comment wasn't really a deal breaker for me, but when I first read the code for this I was like "what's caller?" Since I believe these check functions were originally basically just static functions.

Reply:
Sounds good I’ll clean it up and add the other checks.